desktop/app.js

```diff
@@ -52,6 +53,19 @@ module.exports = function main() {
       show: false,
     });
 
+    // Tray
+    const tray = new Tray(iconPath)
+
+    tray.on('click', () => {
+      mainWindow.isVisible() ? mainWindow.hide() : mainWindow.show()
+    })
+    mainWindow.on('show', () => {
+      tray.setHighlightMode('always')
```

Comment:
what are these highlight modes for? can we add a comment to explain since thy are a bit surprising here in the code?

Reply:
They are visual effects available on macos. Doesn't make a diference on Linux.
More details here: https://github.com/electron/electron/blob/master/docs/api/tray.md#traysethighlightmodemode-macos
I can either remove it or put the comment, what you prefer?

Reply:
Please leave a comment. Oftentimes I raise the question just to hint at the need for a comment in the code so that people will have the context in the future when they read it :wink: